### ISSUE TITLE
Incorrect linux/arm64 image

### ISSUE THREAD

**Issue by bhvngt** [supabase/postgres#143]
# Bug report
## Describe the bug
When I pull `linux/arm64` version of [supabase/postgres](https://hub.docker.com/layers/supabase/postgres/14.1.0/images/sha256-22d7f41a55cb7410d156ae357ce36d8f17d9b62d919e6ab479db9ba2a291c60c?context=explore) and run it as a container, it gives me `x86_64` as platform on executing `uname -a`.

I executed this commands on my `Macbook M1 Pro`.

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. docker pull supabase/postgres:latest
2. docker run --rm -it --platform linux/arm64 --name supabase supabase/postgres:14.1.0@sha256:22d7f41a55cb7410d156ae357ce36d8f17d9b62d919e6ab479db9ba2a291c60c /bin/bash
3. uname -a
4. Output is `Linux 2ee3f3bf12c3 5.10.76-linuxkit #1 SMP PREEMPT Mon Nov 8 11:22:26 UTC 2021 x86_64 GNU/Linux`

## Expected behavior

`uname -a` should give `aarch64` as platform architecture

## Screenshots

![image](https://user-images.githubusercontent.com/79074469/153996333-828a8b8b-5377-4d41-8105-60815fe01e5d.png)

## System information

- OS: Macbook M1 Pro OSX 12.2

## Additional context

Rest of the supabase images such as `supabase/gotrue`, `supabase/postgres-meta` etc pulls correct arch for arm based system.

I have also tried pulling the image by supplying the arm64 digest. It still gives me `x86` arch. While `x86` based image works fine with docker on mac, it runs on a much reduced performance with high cpu load.

I was trying to download these images in linux VM running on top of my M1 Pro. There it completely breaks since the VM does not support running `x86` based images.

**Comment by sarumont** [comment]
Looks like #167 fixed this...any chance of getting a new build pushed soonish? :)

**Comment by sarumont** [comment]
For anyone who needs this now, I have built a version of the image, available under `sarumont/supabase:v14`. Note that `pg_graphql` failed to build for me, so I removed it from the image.

**Comment by soedirgo** [comment]
This is fixed as of 14.1.0.66. Thanks for the patience!

**Comment by johanatan** [comment]
Still an issue:

```
jonathan@Mac-Studio scripts % docker pull supabase/postgres:latest
latest: Pulling from supabase/postgres
Digest: sha256:33d69ef6d6941b411e1e96950735e2844eda6b9477b95160f155e9c6d51d22d6
Status: Image is up to date for supabase/postgres:latest
docker.io/supabase/postgres:latest
jonathan@Mac-Studio scripts % 

jonathan@Mac-Studio scripts % docker run --platform linux/arm64 supabase/postgres  
Unable to find image 'supabase/postgres:latest' locally
latest: Pulling from supabase/postgres
Digest: sha256:33d69ef6d6941b411e1e96950735e2844eda6b9477b95160f155e9c6d51d22d6
Status: Image is up to date for supabase/postgres:latest
docker: image with reference docker.io/supabase/postgres:latest was found but does not match the specified platform: wanted linux/arm64, actual: linux/amd64.
See 'docker run --help'.
jonathan@Mac-Studio scripts % 
```
